java/api/src/main/java/io/ray/api/call/ActorCreator.java

```diff
@@ -20,6 +20,8 @@
   public ActorCreator(RayFuncR<A> func, Object[] args) {
     this.func = func;
     this.args = args;
+    /// Handle Static defined concurrency groups.
+    builder.setConcurrencyGroups(Ray.internal().extractConcurrencyGroups(this.func));
```

Comment:
What will happen if a user defines concurrency groups both statically and dynamically?

Reply:
It's an undefined behavior. From implementation details, static definition will be overwritten as dynamic definition.